Index.html

```diff
@@ -27,5 +27,113 @@ <h4 class="connect">LET'S CONNECT</h4>
       <li><img src="Assests/Vectorangerllist.svg"></li>
     </ul>
   </section>
+  <section class="worksSection">
+    <div class="space"></div>
```

Comment:
- [ ] Nice effort. but there seems to be a missing margin-top from your header to the work section. This does not give a good view or a better user experience. Kindly stick to the design to make it look great 👍 

## Your view:
![image](https://user-images.githubusercontent.com/50990925/235334923-d32d9d7d-f44f-4b00-a5bb-20dfc73bcc7b.png)

## Figma:
![image](https://user-images.githubusercontent.com/50990925/235334946-809e0ef5-2247-4ab4-911a-d4f14b3ffdfa.png)